connectors/v2/ego-fm.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '#playerInfo';
+
+Connector.artistSelector = '#current > span.artist';
+
+Connector.trackSelector = '#current > span.song';
+
+Connector.isPlaying = function () {
```

Comment:
The new window that opens is in a different domain, and I am not running the connector code there.
I am running it on the radio homepage (www.egofm.de) because that's where the current playing track information is. 
In the other window, the information is shown within a auto-refreshing iframe. 

Have a look if you want, www.egofm.de